modules/internal/src/main/scala/client/monixCalls.scala

```diff
@@ -79,4 +80,20 @@ object monixCalls {
             outputObserver
         ))
     )
+
+  private[this] def listenableFuture2Async[F[_], A](
+      fa: => ListenableFuture[A])(implicit F: Async[F], E: ExecutionContext): F[A] =
```

Comment:
I've changed teh `AsyncContext[F]` requirement with an `Async[F]` requirement, which is the same idea.

Reply:
Maybe we can restrict the implicit req to the `Effect` typeclass?

Reply:
`Effect` is an `Async`, so it would be widening. Also, we're not using `runAsync` on the `F`, so we don't need `Effect`, right?